index.d.ts

```diff
@@ -0,0 +1,50 @@
+declare module 'redux-responsive' {
+
+    export interface IBreakPoints {
+        extraSmall: number
+        small: number,
+        medium: number,
+        large: number,
+        extraLarge: number,
+        infinity: number,
+    }
+
+    export interface IBreakPointResults {
+        extraSmall: boolean
+        small: boolean,
+        medium: boolean,
+        large: boolean,
+        extraLarge: boolean,
+        infinity: boolean,
+    }
+
+
+    export interface IBrowser {
+        _responsiveState: boolean
```

Comment:
slightly nit-picky but we noticed you have commas in the previous definitions but not in the ones below this. I'm not sure what the convention is in typescript but I'd like to be consistent here